esmvaltool/diag_scripts/shared/_base.py

```diff
@@ -74,15 +147,14 @@ class ProvenanceLogger:
             record = {
                 'caption': "This is a nice plot.",
                 'statistics': ['mean'],
-                'domain': 'global',
-                'plot_type': 'zonal',
-                'plot_file': '/path/to/result.png',
```

Comment:
Why is this removed? Related to my comment above.

Reply:
See my answer above :wink: